tools/bin/ci_credentials.sh

```diff
@@ -46,3 +46,4 @@ write_standard_creds source-slack-singer "$SLACK_TEST_CREDS"
 write_standard_creds source-stripe-singer "$STRIPE_INTEGRATION_TEST_CREDS"
 write_standard_creds source-twilio-singer "$TWILIO_TEST_CREDS"
 write_standard_creds source-zendesk-singer "$ZENDESK_SECRETS_CREDS"
+write_standard_creds source-intercom-singer "$INTERCOM_SECRETS_CREDS"
```

Comment:
could you put it into the list in alphabetical order? And rename to `INTERCOM_TEST_CREDS` for consistency. Zendesk was named differently by accident.